src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -188,11 +196,10 @@ describe('HmctsGlobalHeaderComponent', () => {
         href: '',
         active: false
       }]);
-      done();
     });
   });
 
-  it('filters out menu items for which the user does not hold the correct role', (done) => {
+  it('filters out menu items for which the user does not hold the correct role', async () => {
```

Comment:
@johnbenjamin-hmcts - see previous comment on `(done)`.

Reply:
Thank you @Daniel-Lam - I have now reverted the changes.